packages/tree-widget/README.md

```diff
@@ -180,3 +180,15 @@ export class TreeWidgetControl extends WidgetControl {
   }
 }
 ```
+
+### Using the TreeWidgetUiItemsProvider
```

Comment:
@calebmshafer @kckst8 

How do we feel about this? Should this be moved up to above the readme section for ui-1.0? Do we want to remove the old way from the readme?

We need to start standardizing how we document these Extensions/UiProivders

Reply:
I think we remove the 1.0 example entirely. Nobody should be using that any longer